aikau/src/main/resources/alfresco/notifications/templates/AlfNotification.html

```diff
@@ -1,5 +1,6 @@
 <div class="alfresco-notifications-AlfNotification">
    <div class="alfresco-notifications-AlfNotification__container">
+      <div class="alfresco-notifications-AlfNotification__close" data-dojo-attach-event="click:_onCloseClick">&times;</div>
```

Comment:
This is BEM, which is definitely intended. We use double-underscore to delineate the OO model of the code and double-hyphen to indicate states. It should not be embedding behaviour, but instead annotating by state and item and leaving the CSS to determine how it looks and the JS to determine what it does.